_posts/2022-02-23-zero-six-two-release.md

```diff
@@ -49,8 +49,8 @@ We'll be writing more about how we integrated with `rich` on the blog in the com
 As the codebase and functionality of ZenML grows, we always want to make sure [our documentation](https://docs.zenml.io/) is clear, up-to-date and easy to use. We made a number of changes in this release that will improve your experience in this regard:
 
 - Ensure *quickstart* example code is identical across everywhere it is referenced.
-- Added core concepts back into the [main glossary](https://docs.zenml.io/reference/glossary) (sorted alphabetically and made concise).
-- Added [cloud-specific guide](https://docs.zenml.io/v/0.6.1/features/cloud-pipelines/guide-aws-gcp-azure) for deploying pipelines.
+- Added core concepts back into the [main glossary](https://docs.zenml.io/) (sorted alphabetically and made concise).
+- Added [cloud-specific guide](https://docs.zenml.io/) for deploying pipelines.
```

Comment:
kinda useless to adjust these, maybe we should not touch the old release blog posts anyways, IMO it's kinda expected the links in those would be outdated and I don't think anyone cares about them anymore anyways

Reply:
Actually, maybe we should even remove the old release blog posts altogether

Reply:
@htahir1 @strickvl might have opinions on this though ;)